package.json

```diff
@@ -14,7 +14,7 @@
   },
   "bugs": "https://github.com/commercetools/orders-update/issues",
   "license": "MIT",
-  "main": "dist/index.js",
+  "main": "dist/orders-update.js",
```

Comment:
Prefer `dist`, more obvious naming I think.

Currently our tools seem to use `dist` or `lib` pretty 50/50, so we can decide which to go with from now on:
- https://github.com/sphereio/sphere-node-cli/blob/master/package.json#L24
- https://github.com/sphereio/sphere-product-type-export/blob/master/package.json#L5
- https://github.com/sphereio/sphere-category-sync/blob/master/package.json#L27
- https://github.com/sphereio/sphere-stock-import/blob/master/package.json#L33
- https://github.com/sphereio/sphere-product-type-import/blob/master/package.json#L5
- https://github.com/sphereio/sphere-order-export/blob/master/package.json#L42